### ISSUE TITLE
Full screen is broken??

### ISSUE THREAD

**Issue by meelijane** [OscarGodson/EpicEditor#331]

**Comment by OscarGodson** [comment]
Do you have anymore details? Browser, OS, code, screenshots, anything?

**Comment by meelijane** [comment]
Yep! Sorry, here they are:

Chrome Version 35.0.1916.153

These screenshots are from the Epic Editor website.
https://www.dropbox.com/s/0n7dpfxp2621sf2/Screenshot%202014-06-15%2022.02.32.png
https://www.dropbox.com/s/t2fesd5i6bpy33t/Screenshot%202014-06-15%2022.02.35.png

![screenshot 2014-06-15 22 02 32](https://cloud.githubusercontent.com/assets/1193125/3280906/4aa329f2-f485-11e3-85fe-f543df107d99.png)
![screenshot 2014-06-15 22 02 35](https://cloud.githubusercontent.com/assets/1193125/3280907/4aa63426-f485-11e3-896e-11c512435598.png)

**Comment by zethussuen** [comment]
@meelijane can you also provide info on OS? Also open your inspector console and click full screen. Do you have an issues?

**Comment by meelijane** [comment]
It looks like it's resolved on your site now, but we uninstalled it for ours.

**Comment by sscovil** [comment]
@OscarGodson @zethussuen I am experiencing the same thing in Chrome (Version 35.0.1916.153) and Safari (Version 7.0.4) on Mac OSX (Version 10.9.3).

In Chrome, the full screen view is blank like the screenshots @meelijane shared. In Safari, it never switches to full screen mode but the editor disappears from the DOM. In both browsers, the editor buttons for preview and full screen do not show up in the editor, but three small squares (empty buttons?) do appear in the upper-left on hover.

Also, I have had one user complain about FireFox crashing while editing in full screen mode. I don't have any useful details about that and have not been able to reproduce it.

**NOTE:** This is a fairly new issue, because when I first made the decision to use EpicEditor in my project about ~2 months ago, it worked perfectly in Chrome.

**Comment by meelijane** [comment]
Not sure if this is useful but at some point between now and when I reported it, I saw EpicEditor working properly, and then it returned to the blank screen.

**Comment by zethussuen** [comment]
I'm able to reproduce locally on Chrome 35.0.1916.153. Looking into whether it's a Full Screen API bug. Thanks for the reports.

**Comment by meelijane** [comment]
Yeah, I suspected that's what it was too. Good luck and thanks for looking in to it :)

**Comment by mikebaldry** [comment]
Ive just found this after installing, full screen mode _sometimes_ just displays white - if you keep closing and opening it does display eventually and works as expected

OS X Yosemite, Chrome Version 39.0.2171.95 (64-bit)
